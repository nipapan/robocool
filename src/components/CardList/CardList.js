import React from 'react';
import Card from '../Card/Card';

const CardList = ({robots}) => {
    return (
       <div>
          {robots.map((robot) => <Card key={robot.id} robot={robot} />)}
       </div>
    );
}

export default CardList;
