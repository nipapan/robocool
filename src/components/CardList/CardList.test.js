import {shallow} from 'enzyme';
import React from 'react';
import CardList from "./CardList";

const mockRobots = [
   {
      id: 1,
      name: 'Palo',
      email: 'palow@gmail.com'
   },
   {
      id: 2,
      name: 'Tom',
      email: 'tom@gmail.com'
   }
]

it('renders CardList', () => {
   expect(shallow(<CardList robots={mockRobots}/>)).toMatchSnapshot();
})