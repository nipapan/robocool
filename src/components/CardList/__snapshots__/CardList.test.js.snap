// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders CardList 1`] = `ShallowWrapper {}`;
